app/app/ssg/product-gated/page.tsx

```diff
@@ -1,12 +1,58 @@
+"use client";
+
 import { NextAppPage } from "@/types/app-dir";
+import Head from "next/head";
+import styles from "../../../../styles/Home.module.css";
+import { Inter } from "next/font/google";
+
+const inter = Inter({ subsets: ["latin"] });
 
 /**
  * this page is protected by the middleware, so if
  * no ssg-bailouts are used in here this page will
  * be statically served
  */
 const Page: NextAppPage = () => {
-  return <>Product owned!</>;
+  return (
+    <>
+      <div className={styles.container}>
+        <Head>
+          <title>Paywall App</title>
+          <meta name="description" content="Generated by create next app" />
+          <link rel="icon" href="/logo.svg" />
+        </Head>
```

Comment:
the next.js app directory uses a special metadata export for setting these tags.
see: https://beta.nextjs.org/docs/api-reference/metadata

Reply:
you might have to update the version of next we use to enable this feature, its been introduced in 13.2